### PR TITLE
Replaced optimist with yargs

### PR DESCRIPTION
**Pull request by Ratstail91**
This is, admittedly just a drop-in replacement, but this issue pops up in literally every project I have, and it's driving me nuts.

### CODE REVIEW

File: .github/workflows/node.js.yml [hunk 1]
```diff
@@ -0,0 +1,29 @@
+# This workflow will do a clean install of node dependencies, build the source code and run tests across different versions of node
```
Comment (kibertoad) [inline .github/workflows/node.js.yml:1]: Is there a reason why this is needed if there is already CircleCI workflow in place?

File: package.json [hunk 2]
```diff
@@ -29,11 +29,11 @@
     "nconf": "^0.10.0",
     "nssocket": "^0.6.0",
     "object-assign": "^4.1.1",
-    "optimist": "^0.6.1",
```
Comment (kibertoad) [inline package.json:32]: Can you remove reference to `optimist` from `cli.js` as well?

**Comment by kibertoad** [comment]
Looks good, thanks a lot! Could you address a couple of very minor comments?

File: .github/workflows/node.js.yml (hunk 1 above)
Comment (Ratstail91) [reply .github/workflows/node.js.yml:1]: I've never used circleCI before, I added this out of habit - feel free to remove this if it's redundant.

File: package.json (hunk 2 above)
Comment (Ratstail91) [reply package.json:32]: Done! Sorry for the slow response.

**Comment by Ratstail91** [comment]
Sure! I've addressed the comments. I can remove the redundant testing if needed, otherwise she's good to go!

**Comment by kibertoad** [comment]
@Ratstail91 Could you please remove the GA part of the PR? I can remove it myself later, but won't have time for this today.

**Comment by Ratstail91** [comment]
@kibertoad All done! Sorry for the delay!

**Comment by kibertoad** [comment]
Thanks a lot!

**Comment by hc-12** [comment]
Thanks for the fix guys, can we have this as patch release soon?

**Pull request merged**